### PR TITLE
Don't Count set_color or alpha blocks

### PR DESCRIPTION
**Pull request by Hamms**
We want to allow students to use `set_color` and `alpha` blocks without
being penalized, since these are purely aesthetic operations.

[Fixes #117260121](https://www.pivotaltracker.com/n/projects/1256652/stories/117260121)

### CODE REVIEW

File: apps/src/feedback.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ var HINT_REQUEST_PLACEMENT = {
   RIGHT: 2  // Hint request button is on right.
 };
 
+// Types of blocks that do not count toward displayed block count. Used
+// by FeedbackUtils.blockSHouldBeCounted_
```
Comment (islemaster) [inline apps/src/feedback.js]: `s/blockSHould/blockShould`

**Comment by islemaster** [comment]
:+1: LGTM.  :tea:

**Comment by Hamms** [comment]
could you give this another quick look? I realized that I forgot to adjust the hardcoded ideal block counts for our legacy artist levels to reflect the new logic.

Note that as Kiki has been revisiting existing levels in CSF, she's been updating ideal block counts with the assumption that these blocks are not counted.

**Comment by islemaster** [comment]
Looks good.

**Pull request merged**